sympy/plotting/plot.py

```diff
@@ -908,176 +943,195 @@ def __init__(self, parent):
+                self.ax[i].spines['left'].set_position('zero')
+                self.ax[i].spines['right'].set_color('none')
+                self.ax[i].spines['bottom'].set_position('zero')
+                self.ax[i].spines['top'].set_color('none')
+                self.ax[i].spines['left'].set_smart_bounds(True)
+                self.ax[i].spines['bottom'].set_smart_bounds(False)
+                self.ax[i].xaxis.set_ticks_position('bottom')
+                self.ax[i].yaxis.set_ticks_position('left')
 
     def process_series(self):
         parent = self.parent
-
-        for s in self.parent._series:
-            # Create the collections
-            if s.is_2Dline:
-                collection = self.LineCollection(s.get_segments())
-                self.ax.add_collection(collection)
-            elif s.is_contour:
-                self.ax.contour(*s.get_meshes())
-            elif s.is_3Dline:
-                # TODO too complicated, I blame matplotlib
-                mpl_toolkits = import_module('mpl_toolkits',
-                    __import__kwargs={'fromlist': ['mplot3d']})
-                art3d = mpl_toolkits.mplot3d.art3d
-                collection = art3d.Line3DCollection(s.get_segments())
-                self.ax.add_collection(collection)
-                x, y, z = s.get_points()
-                self.ax.set_xlim((min(x), max(x)))
-                self.ax.set_ylim((min(y), max(y)))
-                self.ax.set_zlim((min(z), max(z)))
-            elif s.is_3Dsurface:
-                x, y, z = s.get_meshes()
-                collection = self.ax.plot_surface(x, y, z,
-                    cmap=getattr(self.cm, 'viridis', self.cm.jet),
-                    rstride=1, cstride=1, linewidth=0.1)
-            elif s.is_implicit:
-                #Smart bounds have to be set to False for implicit plots.
-                self.ax.spines['left'].set_smart_bounds(False)
-                self.ax.spines['bottom'].set_smart_bounds(False)
-                points = s.get_raster()
-                if len(points) == 2:
-                    #interval math plotting
-                    x, y = _matplotlib_list(points[0])
-                    self.ax.fill(x, y, facecolor=s.line_color, edgecolor='None')
+        if isinstance(self.parent, Plot):
+            series_list = [self.parent._series]
+        else:
+            series_list = self.parent._series
+
+        for i, (series, ax) in enumerate(zip(series_list, self.ax)):
+            for s in series:
+                 # Create the collections
+                if s.is_2Dline:
+                    collection = self.LineCollection(s.get_segments())
+                    ax.add_collection(collection)
+                elif s.is_contour:
+                    ax.contour(*s.get_meshes())
+                elif s.is_3Dline:
+                     # TODO too complicated, I blame matplotlib
+                    mpl_toolkits = import_module('mpl_toolkits',
```

Comment:
Similar here. If import_module throws an exception, maybe one should catch it and return a sensible message?